.github/workflows/pre_merge.yml

```diff
@@ -18,26 +18,30 @@ concurrency:
 
 jobs:
   Code-Quality-Checks:
-    runs-on: [self-hosted, linux, x64, dev]
+    runs-on: ubuntu-20.04
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: 3.8
       - name: Install dependencies
         run: python -m pip install tox
       - name: Code Quality Checks
-        run: tox -re pre-commit
+        run: tox -e pre-commit
```

Comment:
Is it okay when someone changes the requirements?

Reply:
AFAIK, the dependency changes will re-create env and -r is an option for the "force re-create"

Reply:
Thanks !